docs/css/colors-collection.css

```
@@ -0,0 +1,46 @@
+/* Container */
+
+.color-palette {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 20px;
+  margin-bottom: 45px;
+}
+
+.gradient-collection {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 128px);
+  grid-gap: 16px;
+  margin-top: 35px;
+}
+
+/* Color Swatch */
+
+.color-palette__swatch {
+  width: 72px;
+  height: 72px;
+  padding-top: 26px;
+  text-align: center;
+  font: var(--font-primary);
+}
+
+.color-palette__swatch:first-child {
+  border-top-left-radius: 8px;
+  border-bottom-left-radius: 8px;
+}
+
+.color-palette__swatch:last-child {
+  border-top-right-radius: 8px;
+  border-bottom-right-radius: 8px;
+}
+
+/* Gradient Swatch */
+
+.gradient-collection__swatch {
+  border-radius: 8px;
```

```suggestion
  border-radius: var(--swatch-radius);
```

This `8px` value repeats a few times. Do you think it's worth creating a variable to facilitate future maintenance?

---

Unsure 🤔 I think it's unlikely that the border radius of the swatches would be adjusted...